gradient_descent.ipynb

```diff
@@ -198,7 +198,7 @@
     "\n",
     "kwargs_list = [\n",
     "    dict(\n",
-    "        point=gd.rand_point(ranges, ID=i, padding=0.1),\n",
+    "        point=gd.rand_point(ranges, ID=i, padding=0.001),\n",
```

Comment:
You should use very small padding when generating random points, otherwise the edge of the control parameter space wont be examined. E.g. if you examine R_E form 1.0 to 500.0 um, padding=0.1 would translate to 50 um, so random points have R_E between 51.0 and 450.0 um. You migth have the optimum at 38.0 um